companies/digitalo.json

```diff
@@ -0,0 +1,16 @@
+{
+    "slug": "digitalo",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Re-In Retail International GmbH",
+    "address": "Re-In Retail International GmbH\nNordring 98a\n90409 Nürnberg\nDeutschland",
+    "phone": "+49 911 27430933",
+    "fax": "+49 911 27430934",
```

Comment:
The privacy policy lists a phone and fax number specific to privacy-related requests. Please use that instead ([see](https://github.com/datenanfragen/data/#data-sources)).

Reply:
https://www.digitalo.de/Datenschutz.html
![image](https://user-images.githubusercontent.com/30869493/135644770-119c0821-3fd8-4efc-8da5-719185c73c21.png)

Reply:
Under "Recht auf Information/ Widerspruch", it says: "Bei Fragen zur Erhebung, Verarbeitung oder Nutzung Deiner personenbezogenen Daten, bei Auskünften, Berichtigung, Sperrung oder Löschung von Daten sowie Widerruf erteilter Einwilligungen oder Widerspruch gegen eine bestimmte Datenverwendung wende Dich bitte direkt an" with other contact details.